MediaBrowser.Controller/Entities/Folder.cs

```diff
@@ -421,13 +421,12 @@ private async Task ValidateChildrenInternal2(IProgress<double> progress, bool re
                     validChildren.Add(child);
                 }
 
+                // That's all the new and changed ones - now see if there are any that are missing
+                var itemsRemoved = currentChildren.Values.Except(validChildren).ToList();
                 var shouldRemove = !IsRoot || allowRemoveRoot;
                 // If it's an AggregateFolder, don't remove
```

Comment:
This comment needs updating as the behavior is definitely different

Reply:
I think perhaps "that need to be removed" is more correct?